quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/controller/MainMenuController.java

```diff
@@ -17,15 +17,17 @@
 public class MainMenuController extends AbstractController {
 	
 	@FXML
-	private Button exitButton;
+	private Button button;
```

Comment:
This actually is the exit button. Why do you think changing it to a less clear name is better?

Reply:
because the same so called "exitButton" is in use in other places  of the class regardless of its use, and nothing is special about the exit button in order to assign a special variable with unique naming